pkg/cmd/cli/cmd/create/policy_binding.go

```diff
@@ -0,0 +1,129 @@
+package create
+
+import (
+	"fmt"
+	"io"
+
+	"github.com/spf13/cobra"
+
+	"k8s.io/kubernetes/pkg/api/meta"
+	cmdutil "k8s.io/kubernetes/pkg/kubectl/cmd/util"
+	"k8s.io/kubernetes/pkg/runtime"
+
+	authorizationapi "github.com/openshift/origin/pkg/authorization/api"
+	"github.com/openshift/origin/pkg/client"
+	"github.com/openshift/origin/pkg/cmd/util/clientcmd"
+)
+
+const (
+	PolicyBindingRecommendedName = "policybinding"
+
+	policyBindingLong = `
+Create a policy binding that references the policy in the targetted namespace.`
+
+	policyBindingExample = `  # Create a policy binding in namespace "foo" that references the policy in namespace "bar"
+  $ %[1]s bar -n foo`
+)
+
+type CreatePolicyBindingOptions struct {
+	BindingNamespace string
+	PolicyNamespace  string
+
+	BindingClient client.PolicyBindingsNamespacer
+
+	Mapper       meta.RESTMapper
+	OutputFormat string
+	Out          io.Writer
+	Printer      ObjectPrinter
+}
+
+type ObjectPrinter func(runtime.Object, io.Writer) error
+
+// NewCmdCreateServiceAccount is a macro command to create a new service account
+func NewCmdCreatePolicyBinding(name, fullName string, f *clientcmd.Factory, out io.Writer) *cobra.Command {
+	o := &CreatePolicyBindingOptions{Out: out}
+
+	cmd := &cobra.Command{
+		Use:     name + " TARGET_POLICY_NAMESPACE",
+		Short:   "Create a policy binding that references the policy in the targetted namespace.",
+		Long:    policyBindingLong,
+		Example: fmt.Sprintf(policyBindingExample, fullName),
+		Run: func(cmd *cobra.Command, args []string) {
+			cmdutil.CheckErr(o.Complete(cmd, f, args))
+			cmdutil.CheckErr(o.Validate())
+			cmdutil.CheckErr(o.CreatePolicyBinding())
```

Comment:
@soltysh ah. lol.

I'm amazed they do:
```go
      if err := opts.Complete(f, cmd, args, out); err != nil {
        cmdutil.CheckErr(err)
      }
```

since `cmdutil.CheckErr` handles the `nil` case itself.